modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/dashboard-sales/index.css

```
@@ -30,6 +30,18 @@
 	width: 19px;
 }
 
+.donut-chart-space-legend {
```

Please change it here and change it everywhere else that makes use of it.
```suggestion
.dashboard-sales-space-legend {
```

---

Done.